wrappers/models_configurations/chatgpt_config.py

```diff
@@ -0,0 +1,13 @@
+from .base_config import BaseConfiguration
+from .config_types import ConfigTypes
+from typing import TypeVar, Literal
+
+chatGptModel = TypeVar('chatGptModel', bound=Literal["gpt-4", "gpt-4-0314",
+                                                      "gpt-4-32k", "gpt-4-32k-0314",
+                                                        "gpt-3.5-turbo", "gpt-3.5-turbo-0301"])
+
+class GPTConfiguration(BaseConfiguration):
+    def __init__(self, api_key: str, model: chatGptModel) -> None:
+        super().__init__(api_key)
+        self.config_type = ConfigTypes.CHATGPT
```

Comment:
do we need that property?

Reply:
We use that property when we dump the configuration into a file.
look at `wrappers\models_configurations\base_config`, at the `to_file` function

Reply:
got it, it seems this property only purpose is for the saved file on the computer, I think the name should reflect this intent. By stating a specific name like - "config_file_name" (or better) the user can know what to expect

Reply:
Makes sense, I liked the second suggestion with the abstract method and went with it